packages/api/proto/ibc/applications/transfer/v1/genesis.proto

```diff
@@ -1,19 +0,0 @@
-syntax = "proto3";
-
-package ibc.applications.transfer.v1;
```

Comment:
The latest proto from the ibc repository was causing errors when generating code due to an import that does not exist. I think I found the right file but received additional errors. Did not update ibc proto for now and maybe best to address as a separate issue.

Reply:
the issue is that we do use ibc in the registry app, in particular https://buf.build/cosmos/ibc/docs/main:ibc.applications.transfer.v1#ibc.applications.transfer.v1.Query.DenomTrace and https://buf.build/cosmos/ibc/docs/main:ibc.applications.transfer.v1#ibc.applications.transfer.v1.Query.DenomTraces

Reply:
I can add back the proto files that are in the current api package and then maybe we can address updating the ibc proto files in a separate issue. There are probably a few issues we need to resolve before this new version of the api package is ready for regen-web but hopefully those won't be too difficult to address in followups.

Reply:
proto https://github.com/regen-network/regen-js/pull/68/commits/02115847145be78f23d53f6872f6eb322ff396a6 and generated https://github.com/regen-network/regen-js/pull/68/commits/7bfebc81957751ee7075144a5c8ea4f31d17570a